com.unity.render-pipelines.core/Runtime/Common/GlobalDynamicResolutionSettings.cs

```diff
@@ -102,5 +104,11 @@ public static GlobalDynamicResolutionSettings NewDefault() => new GlobalDynamicR
         public bool forceResolution;
         /// <summary>The resolution percentage forced in case forceResolution is set to true.</summary>
         public float forcedPercentage;
+
+        /// <summary>Enable to support lowResTransparencyMinimumThreshold</summary>
+        public bool enableLowResTransparencyThreshold;
```

Comment:
I am actually now wondering if we need the kill switch, maybe just having the threshold is enough? This way 0.0f is equivalent to disable the threshold? Not sure, what do you think? Might make the UI less cumbersome.

Reply:
yep killing this